cli/src/main.rs

```diff
@@ -34,19 +34,6 @@ struct Args {
         num_args(0..=1),
     )]
     terminal_colors: bool,
-    /// Whether the current peer should submit the genesis block or not
-    ///
-    /// Only one peer in the network should submit the genesis block.
-    ///
-    /// This argument must be set alongside with `genesis.file` and `genesis.private_key`
-    /// configuration options. If not, Iroha will exit with an error.
-    ///
-    /// In case when the network consists only of this one peer, i.e. the amount of trusted
-    /// peers in the configuration (`sumeragi.trusted_peers`) is less than 2, this peer must
-    /// submit the genesis, since there are no other peers who can provide it. In this case, Iroha
-    /// will exit with an error if `--submit-genesis` is not set.
-    #[arg(long)]
-    submit_genesis: bool,
```

Comment:
Why this is removed? Do we assume that every peer should have genesis block provided in file?

Reply:
I thought that initially (https://github.com/hyperledger/iroha/issues/4225#issuecomment-1953390635) but now judging by the discussion in this pr I might restore this option

Reply:
Rolled back these changes